### ISSUE TITLE
cannot import name 'USE_RTLD_GLOBAL_WITH_LIBTORCH' on when importing torch

### ISSUE THREAD

**Issue by idanh** [pytorch/pytorch#43939]
## 🐛 Bug

Error `cannot import name 'USE_RTLD_GLOBAL_WITH_LIBTORCH'` after doing `import torch` on a fresh pip install.

## To Reproduce

Using AWS SageMaker Notebook with Jupyter Lab, running:

`!pip install torch torchvision`

And after installation is completed successfully, the following error appeared in my notebook:
```
ImportError                               Traceback (most recent call last)
<ipython-input-27-eb42ca6e4af3> in <module>
----> 1 import torch

~/anaconda3/envs/python3/lib/python3.6/site-packages/torch/__init__.py in <module>
     19 
     20 from ._utils import _import_dotted_name
---> 21 from ._utils_internal import get_file_path, prepare_multiprocessing_environment, \
     22     USE_RTLD_GLOBAL_WITH_LIBTORCH, USE_GLOBAL_DEPS
     23 from .version import __version__

ImportError: cannot import name 'USE_RTLD_GLOBAL_WITH_LIBTORCH'
```

## Expected behavior

Ability to import and use pytourch

## Environment

PyTorch version: 1.6.0
Is debug build: False
CUDA used to build PyTorch: 10.2

OS: Amazon Linux AMI 2018.03 (x86_64)
GCC version: (GCC) 7.2.1 20170915 (Red Hat 7.2.1-2)
Clang version: Could not collect
CMake version: version 3.13.3

Python version: 3.6 (64-bit runtime)
Is CUDA available: False
CUDA runtime version: 10.0.130
GPU models and configuration: Could not collect
Nvidia driver version: Could not collect
cuDNN version: Probably one of the following:
/usr/local/cuda-10.1/targets/x86_64-linux/lib/libcudnn.so.7.6.5
/usr/local/cuda-10.2/targets/x86_64-linux/lib/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip3] numpy==1.18.1
[pip3] numpydoc==0.9.2
[conda] blas                      1.0                         mkl  
[conda] mkl                       2020.0                      166  
[conda] mkl-service               2.3.0            py37he904b0f_0  
[conda] mkl_fft                   1.0.15           py37ha843d7b_0  
[conda] mkl_random                1.1.0            py37hd6b4f25_0  
[conda] numpy                     1.18.1           py37h4f9e942_0  
[conda] numpy-base                1.18.1           py37hde5b4d6_1  
[conda] numpydoc                  0.9.2                      py_0

cc @ezyang @seemethere @malfet

**Comment by ezyang** [comment]
This looks like some sort of botched installation situation. If you `pip uninstall torch` twice and then try reinstalling what happens? CAn you post the full install log for `pip install torch torchvision`?

**Comment by idanh** [comment]
Hi @ezyang 
Maybe that was the case, as I started the same notebook this morning (no touch installation):

<img width="1208" alt="Screen Shot 2020-09-02 at 10 55 29" src="https://user-images.githubusercontent.com/614496/91954710-d9567800-ed0a-11ea-97e9-f3dcc4d470f3.png">

For some background I tried uninstalling and re-installing yesterday the same way you suggested, to no eval.
Am not exactly sure what happened, but as it seems to all work fine right now I'll close that ticket with this comment.

Thanks for taking a look, will update if stumble on the same issue again + pip logs.
